besu/src/main/java/org/hyperledger/besu/cli/options/stable/DataStorageOptions.java

```diff
@@ -143,13 +143,6 @@ public static DataStorageOptions create() {
    */
   public void validate(final CommandLine commandLine, final SyncMode syncMode) {
     if (DataStorageFormat.BONSAI == dataStorageFormat && bonsaiLimitTrieLogsEnabled) {
-      if (SyncMode.FULL == syncMode) {
```

Comment:
why we are removing this one ?

Reply:
This is going to get refactored into a separate PR (see Gary's comment https://github.com/hyperledger/besu/pull/7140#issuecomment-2192593451).

But the reason for removing it is I've changed the logic to set `--bonsai-limit-trie-logs-enabled=false` if `--sync-mode=FULL`, rather than force the user to set `--bonsai-limit-trie-logs-enabled=false`. See https://github.com/hyperledger/besu/blob/7bd09e3ee2ca05298469e7020b037091f3777931/besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java#L2251.

It's a bit like how we automatically set `--tx-pool-price-bump=0` if `--min-gas-price=0`

Reply:
Ok thanks for the info 🙏